truss/templates/trtllm-briton/src/engine.py

```diff
@@ -176,6 +188,32 @@ async def predict(self, model_input):
         if prompt is None and "messages" in model_input:
             messages = model_input.pop("messages")
             prompt = self._tokenizer.apply_chat_template(messages, tokenize=False)
+        if not prompt:
+            raise HTTPException(status_code=400, detail="Prompt cannot be empty.")
+
+        self.validate_input(model_input)
+
+        schema_hash = None
```

Comment:
I'm thinking that the schema generation should actually be in the preprocess method of the truss? it all happens on CPU, so we wouldn't want it to be limited by the predict concurrency